### ISSUE TITLE
Does this library support multiple Sketchpad

### ISSUE THREAD

**Issue by uoau** [Agamnentzar/ag-psd#43]

**Comment by Agamnentzar** [comment]
What is "multiple Sketchpad" ?

**Comment by uoau** [comment]
![image](https://user-images.githubusercontent.com/41247996/105506908-899e3a80-5d05-11eb-9d7d-56170c2c10ff.png)

**Comment by Agamnentzar** [comment]
Oh I see, it seems to be a feature in latest version of Photoshop, sorry right now there's no support for it in the library.

**Comment by uoau** [comment]
Please,I have another question. How to save 'rectangle\round\rounded rectangle', I use `writePsdBuffer function` to export .psd file, but But all the  'rectangle\round\rounded rectangle' became shapes.I want to keep their original shape information.
![image](https://user-images.githubusercontent.com/41247996/105579497-16a6c980-5dc2-11eb-860b-5bebb63eebea.png)
![image](https://user-images.githubusercontent.com/41247996/105579533-4229b400-5dc2-11eb-9d4b-62fa69e9f568.png)

`
    const buffer = fs.readFileSync('./round.psd');
    const psdSource = agPsd.readPsd(buffer, { skipLayerImageData: true, skipCompositeImageData: true, skipThumbnail: true });
    const newBuffer = agPsd.writePsdBuffer(psdSource);
    fs.writeFileSync('new-round.psd', newBuffer);
`

**Comment by Agamnentzar** [comment]
Can you send me PSD file that exhibits this issue, I'll check what's wrong with it.

**Comment by uoau** [comment]
https://hlgcdn.oss-cn-hangzhou.aliyuncs.com/hlg-ui/1611416710663504/8.psd
https://hlgcdn.oss-cn-hangzhou.aliyuncs.com/hlg-ui/1611416737817391/88.psd
How kind of you

**Comment by uoau** [comment]
the round of 8.psd is round, but the round of 88.psd is shape

**Comment by uoau** [comment]
```
        const buffer8 = fs.readFileSync(path.join(__dirname, '8.psd'));
        const psdSource8 = agPsd.readPsd(buffer8, { skipLayerImageData: true, skipCompositeImageData: true, skipThumbnail: true });
        const newBuffer8 = agPsd.writePsdBuffer(psdSource8);
        fs.writeFileSync(path.join(__dirname, '88.psd'), newBuffer8);
```

**Comment by Agamnentzar** [comment]
I added missing support for some sections, you can try if it works now. This change was published as version `11.3.0`

**Comment by uoau** [comment]
How kind of you !!! thanks your help!
But there are still some bug. the 99.psd file is exported form 9.psd, but the `rounded rectangle/smart image` of 99.psd file has some problems, ps will popup a error dialog when i wan to edit it. and some line in the `vector shape` of 99.psd is loss 
https://hlgcdn.oss-cn-hangzhou.aliyuncs.com/hlg-ui/1611443435398564/9.psd
https://hlgcdn.oss-cn-hangzhou.aliyuncs.com/hlg-ui/1611443468842539/99.psd
![image](https://user-images.githubusercontent.com/41247996/105616822-8ba2ef00-5e14-11eb-82c8-eb6c508c3cdc.png)

**Comment by Agamnentzar** [comment]
Released new version `11.4.0` with support for artboards and some fixes for handling vector layers and smart objects. I found some differences in encoding, in new Photoshop version, that don't have any documentation so I'm not sure if everything will be working correctly now.

**Comment by uoau** [comment]
HOW NICE OF YOU! Thanks for your fast response！It's work!

**Comment by Agamnentzar** [comment]
Version `11.4.0` still had some issues with vector layer in the file you linked, those should now be fixes in version `11.5.1`
